crates/ruma-events/src/room/encryption.rs

```diff
@@ -20,6 +20,10 @@ pub struct RoomEncryptionEventContent {
     /// Must be `m.megolm.v1.aes-sha2`.
     pub algorithm: EventEncryptionAlgorithm,
 
+    /// Whether state events should be encrypted alongside message-like events.
+    #[cfg(feature = "unstable-msc3414")]
+    pub encrypt_state_events: bool,
```

Comment:
I don't see this field in the MSC, I guess this a recent addition? Shouldn't it use an unstable prefix?

Also it likely needs the `#[serde(default, skip_serializing_if = "ruma_common::serde::is_default")]` attribute, otherwise this field becomes mandatory during (de)serialization.

Reply:
I don't feel comfortable merging this without an unstable prefix, it would be surprising that a field from an MSC uses its stable name right away on an existing event type. Maybe you can ask around if an unstable prefix can be assigned to this?

Reply:
> As far as I understand, there's a lot of internal discussion going on atm about writing up a new MSC that clears up MSC3414. This should be a necessary addition.

Yeah. It seems likely that we're going to end up needing a new MSC at some point, that adds things like this, and remove some other stuff that we're *not* doing.

In the meantime... msc3414 is maybe the best placeholder?